public/style.css

```diff
@@ -35,12 +34,15 @@ button {
     font-size: 22px;
     font-weight: 700;
     cursor: pointer;
+    margin: 0 auto;
```

Comment:
I would add a `max-width` or `width: fit-content` so the button doesn't span the entire page.

Reply:
Good idea. I didn't know about `fit-content()`. According to MDN, it's a [combination of min() and max()](https://developer.mozilla.org/en-US/docs/Web/CSS/fit-content). Unfortunately as I learned recently, `min()` isn't supported on all browsers, so I'll stick with `max-width` for now.

Reply:
Added in a new commit.